bridge/client/webrtc.js

```diff
@@ -779,19 +779,16 @@
 
             var isInitiator = description.type == "offer";
             whenPeerHandler(function () {
-                latestLocalDescriptionCallback = function () {
-                    a.signalingState = targetState;
-                    resolve();
-                };
-
                 if (hasNewMediaDescriptions)
                     peerHandler.prepareToReceive(localSessionInfo, isInitiator);
 
                 if (remoteSessionInfo)
                     peerHandler.prepareToSend(remoteSessionInfo, isInitiator);
 
-                if (!hasNewMediaDescriptions)
-                    completeQueuedOperation(latestLocalDescriptionCallback);
+                latestLocalDescriptionCallback = function () {
```

Comment:
I don't think we need the latestLocalDescriptionCallback visible outside this function now since it can complete on it's own.